### PR TITLE
Mailhog

### PR DESCRIPTION
**Pull request by maiklgravi**

### CODE REVIEW

File: resources/views/contacts/contactUs.blade.php [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            <form action="{{ route('contactUs.send') }}" method="POST" name="contact-form">
+                <div class="form-row">
+                    <div class="form-group col-md-6">
+                        <label for="email">Email</label>
+                        <input type="email" value="{{ old('email') }}" required="required" class="form-control" name="email" id="email" placeholder="Email">
+                    </div>
+                    <div class="form-group col-md-6">
+                        <label for="name">Name</label>
+                        <input type="text" value="{{ old('name') }}" required="required" class="form-control" name="name" id="name" placeholder="Nume">
+                        @error('name')
```
Comment (egorbwork) [inline resources/views/contacts/contactUs.blade.php:34]: I think you can remove this one. You have the global one.

File: resources/views/emails/contactUs.blade.php [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
+<body>
+<p>User email :{{$email}}</p>
+<p>Name: {{$name}}</p>
+<p>Subject: {{$subjectMessage}}</p>
+<p>Message:</p>
+<p>{{$messageText}}</p>
+
```
Comment (egorbwork) [inline resources/views/emails/contactUs.blade.php:9]: the hmtl is invalid, body closing tag is missing.

**Review by egorbwork — CHANGES_REQUESTED** [review]
Exists serious mistakes. Is not acceptable at this moment.

File: resources/views/emails/contactUs.blade.php (hunk 2 above)
Comment (maiklgravi) [reply resources/views/emails/contactUs.blade.php:11]: Html fixed.

File: resources/views/contacts/contactUs.blade.php [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+                </div>
+                
+                @csrf
+                <button type="submit" class="btn btn-primary">Send</button>
+            </form>
+        </div>
+    </div>
+
+@endsection
+{{-- <div class="form-group col-md-6">
```
[the remark this answers is not in the document]
Comment (maiklgravi) [reply resources/views/contacts/contactUs.blade.php]: Commet deleted

File: resources/views/contacts/contactUs.blade.php (hunk 1 above)
Comment (maiklgravi) [reply resources/views/contacts/contactUs.blade.php]: I delete this.

File: app/Http/Controllers/ContactUsController.php [hunk 4]
```diff
@@ -2,11 +2,43 @@
...
-    public function index() {
-        return view('contact_us');
+    public function view(): View
+    {
+        return view('contacts.contactUs');
+    }
+
+    public function send(ContactUsRequest $request): RedirectResponse
+    {
+        $collback = function (string $test): string{
```
[the remark this answers is not in the document]
Comment (maiklgravi) [reply app/Http/Controllers/ContactUsController.php]: I remove my fault.

**Review by egorbwork — APPROVED** [review]
Looks now fine. Can be merged.

**Pull request merged**